woudc_data_registry/epicentre/metadata.py

```diff
@@ -128,15 +130,18 @@ def add_metadata(entity, dict_):
                 'station_id': station_id,
                 'name': name,
                 'first_seen': date.today()
-            })
+            }, es=False)
 
     c = entity(dict_)
-    REGISTRY.save(c)
+    if psql:
+        REGISTRY.save(c)
+    if es:
+        SEARCH_INDEX.index(entity, c.__geo_interface__)
 
     return c
 
 
-def update_metadata(entity, identifier, dict_):
+def update_metadata(entity, identifier, dict_, psql=True, es=True):
```

Comment:
What's the goal of `psql` flags here?

Reply:
I've been thinking about adding config options to turn on/off certain relations in the registry or indexes in ES under certain conditions. The `es=True` flag had to be added anyways, so it made sense to add both these flags in at once even if one is unused.